keras/backend/mxnet_backend.py

```diff
@@ -2364,8 +2380,8 @@ def switch(condition, then_expression, else_expression):
     # Returns
         The selected tensor.
     """
-    raise NotImplementedError
-
+    condition = eval(condition)
```

Comment:
Should this function have the similar logic as tf.cond:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/control_flow_ops.py#L1676